database/pgsql/pgsql.go

```diff
@@ -114,7 +114,9 @@ func openDatabase(registrableComponentConfig database.RegistrableComponentConfig
 		}
 		src = fmt.Sprintf("%s password=%s", pg.config.Source, password)
 	} else if !os.IsNotExist(err) {
-		log.Errorf("error stating password file %q: %v", passwordFile, err)
+		log.Errorf("pgsql: error stating password file %q; continuing without one: %v", passwordFile, err)
```

Comment:
Why do we continue without a password?

Reply:
DB integration tests do not require a password. We allow Scanner to continue without crashing or anything, but if the database requires it, then it'll fail all connections/queries

Reply:
Yeah, I think this is suboptimal if people mess up and break the scanner password because it'll just spew errors instead of being straightforward in terms of what's wrong. Can we only allow this in a test-focused function?

Reply:
I am ok with us erroring out here for this `else if !os.IsNotExist(err)` since there was some kind of error with the underlying fielsystem/OS; however, for the `else`, I think it's ok to continue because we know no password file was not used.

Also keep in mind this is nothing new. This is just updating the logs for something we already do.

Reply:
I enforce a password in non-tests now